scipy/integrate/_quadrature.py

```diff
@@ -237,13 +238,17 @@ def fixed_quad(func, a, b, args=(), n=5):
     1.0
 
     """
+    xp = array_namespace(a, b, *args)
```

Comment:
Should we pass `args` to `array_namespace`?

Reply:
No, probably not. Originally I was passing `*args` to `xp_promote` and thought better of it. Probably shouldn't pass to `array_namespace` for the same reason.